recipes/object-map.md

```diff
@@ -0,0 +1,73 @@
+import Example from '../components/md/Example';
+
+<Example reactnative>
+
+Defining Object
+```js
+const exampleObject = {
+    type: 'example',
+    value: 3,
+    children: null,
+    isEditable: false
+}
+```
+
+Accessing Object values
+```js
+exampleObject.type;
+exampleObject.value;
+```
+
+</Example>
+
+<Example flutter>
+
+In Dart, there is Map like JS Objects but it is a little bit different
+
+Defining Map
+```dart
+Map<String, dynamic> exampleMap = {
+    'type': 'example',
+    'value': 3,
+    'children': null,
+    'isEditable': false,
+}
+```
+
+Accessing Map values
+```dart
+exampleMap['type'];
+exampleMap['value'];
+```
+
+If you want to define and access Map like JS, you need to define a class
```

Comment:
that's not technically correct, `class` is a user defined type, but `Map` is a data structure, so I'd get rid of this piece of code